.gitlab/pipeline/zombienet/polkadot.yml

```diff
@@ -105,6 +105,14 @@ zombienet-polkadot-functional-0005-parachains-disputes-past-session:
       --local-dir="${LOCAL_DIR}/functional"
       --test="0005-parachains-disputes-past-session.zndsl"
 
+zombienet-polkadot-functional-0006-validator-disabling:
+  extends:
+    - .zombienet-polkadot-common
+  script:
+    - /home/nonroot/zombie-net/scripts/ci/run-test-local-env-manager.sh
```

Comment:
Out of curiosity what is '/home/nonroot/zombie-net/scripts/'? I can't find it in the project.

Reply:
That [script](https://github.com/paritytech/zombienet/blob/main/scripts/ci/run-test-local-env-manager.sh) is part of the zombienet image that we use in gitlab, basically spawn the orchestrator/test-runner that execute the test.